documents/ogc-submission.adoc

```diff
@@ -0,0 +1,156 @@
+
+The submitters agree to abide by the TC Policies and Procedures and OGC Intellectual Property Rights Policy (http://www.opengeospatial.org/ogc/policies) during the processing of this submission.
+
+Once approved, the Community standard Work Item defined by this document is valid for six (6) months.
+
+== Overview of proposed submission
+
+openEO develops an open application programming interface (API) that connects clients in languages such as R, Python, and JavaScript to big Earth observation cloud back-ends in a simple and unified way.
+
+The openEO API specification evolved from the aim to increase the interoperability of big EO data processing of satellite imagery in the cloud. Multiple organiozations agreed to define and implement such an API. Its development was driven by the need to overcome the challenges associated with different tools, APIs, and data formats in geospatial technology. It was developed bottom-up and each version was backed by implementations.
+
+The primary use case is to simplify and unify the data processing using a common API with a set of pre-defined processes. It is beneficial for users as they can still work in their favored programming language without worrying about data organization and pre-processing. They can avoid vendor lock-in as the generated process descriptions can be executed at multiple providers and as such also allows to compare and reproduce processing results more easily.
+
+openEO is an open source project and the specification consists mainly of two parts: The HTTP API description (via OpenAPI) and the pre-defined processes (in JSON). All specifications are released under the Apache 2.0 license. The project is governed by a Project Steering Committee.
+
+openEO is backed by various tools and resources that facilitate its use. These include a variety of server and client implementations, including a web interface for non-programmers and a server implementation that can run locally in a Python environment.
+
+== Relationship to other OGC Standards
+
+The openEO API builds on top of other standards and specifications.
+It's basis is HTTP with JSON as request and response encoding.
+The API specification reuses building blocks from OGC API - Common.
+It also implements the STAC API specification, which is based on the OGC API - Features standard.
+The "Secondary Web Services" in openEO allow to expose a wide-variety of visualization and download services using standards or specifications such as OGC WMS, OGC WMTS, OGC WFS, OGC WCS, OGC API - Tiles, OGC API - Maps, OGC API - Features, XYZ, and more.
+For Authentication the openEO API builds on top of the commonly used standards HTTP Basic and OpenID Connect.
+
+== Alignment with OGC Standards Baseline
+
+openEO continuously tries to align with OGC APIs (especially Common) and STAC. The openEO API standard fits well within the OGC API roadmap. 
```

Comment:
continuously -> omit
tries to align -> aligns

Reply:
```suggestion
openEO aligns with OGC APIs (especially Common) and STAC. The openEO API standard fits well within the OGC API roadmap. 
```